src/order/atoms.lean

```diff
@@ -565,40 +565,120 @@ is_simple_order_iff_is_coatom_bot.trans $ and_congr (not_congr subtype.mk_eq_mk)
 
 end set
 
+namespace order_embedding
+
+variables [partial_order α] [partial_order β]
+
+lemma is_atom_of_map_bot_of_image [order_bot α] [order_bot β] (f : β ↪o α) (hbot : f ⊥ = ⊥) {b : β}
```

Comment:
Is it better to state this in terms of homs that preserve bot? Also, should this be in terms of a `hom_class` rather than order embeddings specifically? @YaelDillies

Reply:
There's no `order_embedding_class`, unfortunately. And although `⊥`-preserving morphisms are a thing in mathlib, they are an implementation detail for stronger morphisms. So I'm fine with the current setup given that it seems to fulfill Jireh's needs.